programs/steward/src/score.rs

```diff
@@ -90,6 +90,7 @@ pub fn validator_score(
     cluster: &ClusterHistory,
     config: &Config,
     current_epoch: u16,
```

Comment:
current_epoch should be u64

Reply:
legacy issue cause of ValidatorHistoryEntry trying to minimize bytes